src/misc/messages.cpp

```diff
@@ -21,39 +21,61 @@
 
 #include "dosbox.h"
 
+#include <clocale>
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
 #include <deque>
-#include <clocale>
+#include <map>
 
 #include "std_filesystem.h"
 #include <string>
 #include <unordered_map>
 
+#include "ansi_code_markup.h"
+#include "checks.h"
 #include "control.h"
 #include "cross.h"
 #include "fs_utils.h"
-#include "string_utils.h"
 #include "setup.h"
+#include "string_utils.h"
 #include "support.h"
-#include "ansi_code_markup.h"
 
 #define LINE_IN_MAXLEN 2048
 
+CHECK_NARROWING();
+
 class Message {
 private:
 	std::string markup_msg = {};
 	std::string rendered_msg = {};
 
+	std::map<uint16_t, std::string> markup_cp_msg   = {};
+	std::map<uint16_t, std::string> rendered_cp_msg = {};
```

Comment:
Does `cp` stand for "code page"? Maybe spell it out? Being new to the code, I don't know what these variables are supposed to represent.

Reply:
Generally, I'm a fan of naming maps like this:

FooByBar
BarToFoo

So you could call tese `markup_msg_by_codepage` and `rendered_msg_by_codepage`. Probably I sound like a broken record, but I value readability over short variable names (within reasonable limits, of course). Especially for non-local variables, such as these class members, I'd prefer more verbose and descriptive names as it greatly aids readability (for people who are unaccustomed to the code, and even for yourself 3-6 months from now 😁 ).

Reply:
Changed. Also changed map names in `unicode.cpp`.